src/v0/destinations/braze/transform.js

```diff
@@ -308,6 +309,37 @@ function processTrackEvent(messageType, message, destination, mappingJson) {
   payload = addMandatoryEventProperties(payload, message);
   payload.properties = properties;
 
+  try {
+    // add,update,remove
+    if (destination.Config.enableNestedArrayOperations) {
+      Object.keys(properties)
+        .filter(k => Array.isArray(properties[`${k}`]))
+        .forEach(key => {
+          // if not specified, send as create attribute
+          if (!properties.nestedOperationType) {
+            attributePayload[key] = properties[key];
+          } else if (
+            supportedOperationTypes.includes(properties.nestedOperationType)
+          ) {
+            attributePayload[key] = {};
+            attributePayload[key][`$${properties.nestedOperationType}`] =
+              properties[key];
+            if (properties.nestedOperationType === "update") {
+              // eslint-disable-next-line no-underscore-dangle
+              attributePayload._merge_objects = isDefinedAndNotNull(
+                properties.mergeObjectsUpdateOperation
+              )
+                ? properties.mergeObjectsUpdateOperation
+                : true;
+            }
+          }
+        });
+      delete properties.nestedOperationType;
+    }
+  } catch (exp) {
+    logger.info("Failure occured during nested array operations", exp);
```

Comment:
```suggestion
    logger.info("Failure occured during nested array operations", exp);
```
Logger is not defined.